webapp/src/css/enketo/medic.less

```diff
@@ -21,16 +21,13 @@
 
     .dropdown-menu .option-wrapper {
       padding: 3px;
-      .option-label {
-        width: calc(100% - 34px);
-      }
```

Comment:
I _think_ this is important, but we'll see what Jenni comes up with.

Reply:
This is part of the "I'm not proposing this". My bad for not making this a multi-line comment.